app/forms/provider_interface/course_wizard.rb

```diff
@@ -10,7 +10,7 @@ class CourseWizard
                   :location_id
 
     validates :course_id, presence: true, on: %i[courses save]
-    validates :study_mode, presence: true, on: %i[study_mode save]
```

Comment:
chatted in person: tests should have failed because of this, investigate why they didn't